.vscode/settings.json

```diff
@@ -13,6 +13,11 @@
   "[typescript]": {
     "editor.formatOnSave": true
   },
+  "[python]": {
+    "editor.formatOnSave": false,
```

Comment:
@gjreda added this configurations to prevent VSCode from automatically format the python code with prettier.
Let me know what is the code formatter we should use and I can configure it instead of "ms-python.python".